### DISCUSSION TITLE
Unnecessary dependencies in Oxalis

### DISCUSSION THREAD

**Discussion by aaron-kumar** [OxalisCommunity/oxalis#520] [Ideas]
Based on observation & discussion, it is found that Oxalis has in some ways become a bit bloated through the years. There are a lot of dependencies and it is not entirely clear whether all of the dependencies are really-2 necessary. Could be an opportunity to clean up a bit and get rid of some dependencies. For example Google Guice (or maybe Guava?) has caused some problems in the past.

**Request to Oxalis Community Member**: Please vote (thumbs up) for this issue if you think this will solve your lots of problems.

**Comment by kukel** [comment]
Going for at least standardized injection annotations makes it easier to use Oxalis as a framework in other broader messaging environments. It is not (too) tightly its own system.
